backend/src/v1/complaint_referral/complaint_referral.service.ts

```diff
@@ -42,6 +48,21 @@ export class ComplaintReferralService {
     // Clear the officer assigend to the complaint
     this._personService.clearAssignedOfficer(createComplaintReferralDto.complaint_identifier);
 
+    // FEATURE FLAG
+    // If both agencies involved have the feature flag active, send the referral email notification
+    const referredByActive = await this._featureFlagService.checkActiveByAgencyAndFeatureCode(
+      createComplaintReferralDto.referred_by_agency_code,
+      "REFEMAIL",
+    );
+    const referrefToActive = await this._featureFlagService.checkActiveByAgencyAndFeatureCode(
+      createComplaintReferralDto.referred_to_agency_code,
```

Comment:
Does this check need to be for both agencies? Why not just the sender?

Reply:
My thought was this would avoid a scenario where a new agency aren't setup for email notifications and the backend starts throwing errors trying to send them anyways. I can pull it for now and if we need the more fine grained control we can add it later.

Reply:
It's not a bad thought if we're going to have future agencies enabled for referrals and added. Might it be better to just raise an exception if there's no email address configured? Something wasn't setup right, throw an error so it gets fixed asap instead of suppressing it?